aptos-move/framework/aptos-framework/doc/multisig_account.md

```diff
@@ -2790,5 +2800,188 @@ Add new owners, remove owners to remove, update signatures required.
 
 </details>
 
+<a id="@Specification_1"></a>
+
+## Specification
```

Comment:
Can you please add the high-level requirements table and the appropriate labels to the .spec.move file?

Reply:
I added the high-level table.